sonar-project.properties

```diff
@@ -0,0 +1,8 @@
+sonar.projectName=sunbird-collection-editor
```

Comment:
This is for the Sonarqube check. This won't break anything in the collection editor. it is just for Sonarcube continuous inspection which will analyze project code on each pull request.